### DISCUSSION TITLE
Batch img2img with caption files - is it possible? how to?

### DISCUSSION THREAD

**Discussion by SAC020** [vladmandic/sdnext#3643] [General]
Hi,

The use case is: I have a bunch of images with associated caption files (like for lora training), and I want to run a batch img2img but instead of having a common prompt for all files in the batch, the prompt for each img2img should be the one from the associated (same filename) caption file.

Is this achievable? If so, how to?

Thanks!

**Comment by vladmandic** [comment]
its not, but its a good feature request and should be relatively easy to implement.
switching this discussion thread to feature request.

**Reply by SAC020** [reply]
thank you!
